modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -1,17 +1,12 @@
-= Refining Batch Steps Processing
+= Using Batch Processing Scopes
```

Comment:
Title change because this section covers more than the Step scope. Also, it does not cover every configuration of the scope but does since it talks about some uses and how to configure them. Not perfect, but I think this page should evolve into an Example/Use Case (big and small) page.

Reply:
One thing is, changing the block size is on the Batch Scope, not the Step or Aggregator. Another is that "Refining" is just not a typical way to present this sort of info.